### PR TITLE
Adding the new openshift-4.6-openstack flavor for OSP-4.6 version

### PR DESCRIPTION
**Pull request by abhis2112**

### CODE REVIEW

File: provision/acc_provision/flavors.yaml [hunk 1]
```diff
@@ -212,6 +212,132 @@ flavors:
     status: null
     hidden: false
     order: 0
```
Comment (snaiksat) [inline provision/acc_provision/flavors.yaml:214]: this needs to be updated to 1 (and all the other order numbers need to be incremented by 1).

File: provision/fix-testdata.sh [hunk 2]
```diff
@@ -0,0 +1,329 @@
+rm -rf /tmp/generated*
```
Comment (snaiksat) [inline provision/fix-testdata.sh:1]: don't commit this file ;-)

File: provision/testdata/list_flavors.stdout.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 INFO: Available configuration flavors:
 INFO: openshift-4.5-openstack:	Red Hat OpenShift Container Platform 4.5 on OpenStack
+INFO: openshift-4.6-openstack:	Red Hat OpenShift Container Platform 4.6 on OpenStack
```
Comment (snaiksat) [inline provision/testdata/list_flavors.stdout.txt:3]: Once you change the order numbers, this order will also change, and you will just need to swap the above two.

**Review by snaiksat — CHANGES_REQUESTED** [review]

File: provision/acc_provision/flavors.yaml (hunk 1 above)
Comment (abhis2112) [reply provision/acc_provision/flavors.yaml:214]: noted

File: provision/fix-testdata.sh (hunk 2 above)
Comment (abhis2112) [reply provision/fix-testdata.sh:1]: ahh yeah, I was careful with that but seems like ended up adding it in multiple places. Resolving..

**Review by apoorva11029 — APPROVED** [review]

**Review by snaiksat — APPROVED** [review]

**Pull request merged**